startlxqtwayland.in

```diff
@@ -164,6 +165,7 @@ elif [ "$COMPOSITOR" = "sway" ]; then
     exec $COMPOSITOR -c $XDG_CONFIG_HOME/lxqt/wayland/lxqt-sway.config
 
 elif [ "$COMPOSITOR" = "Hyprland" ]; then
+    export XDG_CURRENT_DESKTOP="LXQt:$COMPOSITOR:aquamarine"
```

Comment:
Ditto.

Reply:
Yes, it needs https://github.com/lxqt/lxqt-panel/pull/2282

Reply:
With it too.

Reply:
Not from my experience ;)

Reply:
Perhaps because you use your deleted PR. But, in the end, that's no relevant. I'm adding a comment to https://github.com/lxqt/lxqt-panel/pull/2282.